### ISSUE TITLE
Feature Request to Pause a Monitor

### ISSUE THREAD

**Issue by ptgob** [meichthys/uptime_kuma#107]
It would be interesting to be able to pause a monitor from HA (if the Uptime Kuma API allows it). The use case would be to disable a network device (for example via the Unifi integration) and to pause the monitoring of this same device in Uptime Kuma. This would allow for even deeper integration between Uptime Kuma and HA!

Thank you for this integration! :)

**Comment by spitfire** [comment]
Yeah, I'd love it if I could pause monitoring (or even better plan a maintenance for 3 minutes) when I restart Home Assistant.
I could run an automation on certain events and avoid having to start a maintenance window myself or getting notifications that HA is down.

**Comment by andreasbuff** [comment]
Yes, me too. As there is allready to option to set a device in maintenance-mode with Uptime-Kuma, it would be great to have this sensor in the integration as well (e.g. Maintenance, or set Device into deep-sleep, until next WOL, or Cron-Job). 

Thank you for your work!
